streaming-architecture.md

```diff
@@ -1,3 +1,3 @@
 ## Better JSON through streams
```

Comment:
Looks much better. I think there will be conferences where you'll need a longer abstract with more detail, so keep that in mind!

Two minor things:

- [ ] should be `its` versus `it's` (the possessive has no apostrophe)
- [ ] in the `It will explore three frontiers...` sentence I think you need an `and` in front of the last clause: "... server to the client, **and** receiving and processing a stream..."